cloud/pkg/router/listener/message.go

```diff
@@ -76,10 +77,15 @@ func (mh *MessageHandler) HandleMessage(message *model.Message) error {
 		return err
 	}
 	go func(message *model.Message) {
-		_, err := handler(message)
+		resp, err := handler(message)
 		if err != nil {
 			klog.Errorf("handle message occur error, msgID: %s, reason: %s", message.GetID(), err.Error())
 		}
+		if resp != nil {
+			if err = resp.(*http.Response).Body.Close(); err != nil {
+				klog.Errorf("close response occur error, msgID: %s, reason: %s", message.GetID(), err.Error())
+			}
+		}
```

Comment:
plz add it to func httpHandler in cloud\pkg\router\listener\http.go

Reply:
Has the comment been fixed?

Reply:
plz ignore this comment.